src/test/java/net/snowflake/client/jdbc/BindingDataIT.java

```diff
@@ -335,6 +335,7 @@ public void testBindDate(Date dateValue) throws SQLException {
     try (Connection connection = getConnection();
         Statement statement = connection.createStatement()) {
       try {
+        statement.execute("alter session set JDBC_FORMAT_DATE_WITH_TIMEZONE=FALSE");
```

Comment:
why do we need to update tests in this class?

Reply:
Because of the fix for [244](https://github.com/snowflakedb/snowflake-sdks-drivers-issues-teamwork/issues/244), the timezone changed from null to default (null causes it to ignore timezone). In these tests, it checks that the value in the db is the same value pulled from getDate ignoring timezone so I set JDBC_FORMAT_DATE_WITH_TIMEZONE to false.